dcc-repository-core/src/main/java/org/icgc/dcc/repository/core/model/Repositories.java

```diff
@@ -79,9 +79,9 @@ public static Repository getRepository(@NonNull String repoCode) {
   @Getter
   private static final Repository aWSRepository = findRepository(repository -> repository.getSource() == AWS);
   @Getter
-  private static final Repository collabRepository = findRepository(repository -> repository.getSource() == COLLAB_OLD);
+  private static final Repository collabOldRepository = findRepository(repository -> repository.getSource() == COLLAB_OLD);
   @Getter
-  private static final Repository songRepository = findRepository(repository -> repository.getSource() == COLLAB);
```

Comment:
One of the unit tests still has a reference to this. See: `SONGImporterTest.java`

Also that class happens to be incorrectly named (not by you).